tests/src/JIT/SIMD/Vector3Interop.cs

```diff
@@ -71,19 +94,38 @@ public static bool test()
             }
         }
         
+        // Argument passing test for Unix.
+        // Few arguments are passed onto stack.
+        {
+            Vector3 v1 = new Vector3(1, 2, 3);
+            Vector3 v2 = new Vector3(10, 20, 30);
+            float f0 = 0, f1 = 1, f2 = 2, f3 = 3, f4 = 4, f5 = 5, f6 = 6, f7 = 7, f8 = 8;
```

Comment:
Please choose values that are different from 1, 2,3, 10, 20 and 30 to initialize f0 through f8.  If Jitted code is due to a bug is accessing f1, f2 and f3 instead of v1, this test will still pass.